admin/class-plugin-installation.php

```diff
@@ -24,6 +24,23 @@ public function register_hooks() {
 	public function enqueue_scripts() {
 		if ( 'wpseo_licenses' === filter_input( INPUT_GET, 'page' ) ) {
 			$asset_manager = new WPSEO_Admin_Asset_Manager();
+			wp_localize_script(
+				WPSEO_Admin_Asset_Manager::PREFIX . 'plugin-installation',
+				'wpseoPluginInstallationL10n',
+				array(
+					'target'  => array(
+						'domain' => 'https://my.yoast.com',
+						'path'   => '/addsite?url=' . esc_html( home_url() ) . '&plugin=',
+					),
+					'pluginNames' => array(
```

Comment:
I think it's better to have this as a static method in the `WPSEO_Addon_Manager` just to keep all plugin related things in one place.